nemo/collections/nlp/modules/common/megatron/token_level_encoder_decoder.py

```diff
@@ -652,7 +652,11 @@ def forward(
 
                     # speech_logits_list will be used in loss calculation (parallel output)
                     speech_logits_list = []
-                    if self.seq_pattern in ["parallel", "delay_parallel"]:
+                    speech_logits = torch.zeros(
```

Comment:
speech_logits should be an unused variable now, so we can delete this I think.

Reply:
sure.